### DISCUSSION TITLE
Bootstrap5 template not working with docker

### DISCUSSION THREAD

**Discussion by Skyfay** [PrivateBin/PrivateBin#1397] [Q&A] [answered]
### Describe the problem/question

I would like to use the Bootstrap5 template. However, this only works if I select this in the config. 
However, I would like to customize the template, so I did that and then integrated it into the container via docker. Whenever I do this, regardless of whether I select a different name “customtemplate” or overwrite the bootstrap5, I get a white page with no content.
This also happens if I use the template and do not change it. As soon as I integrate it via docker I get no content. 

![image](https://github.com/user-attachments/assets/88e6816b-2dfd-4be5-b8c6-d871da0a6fcb)

```  thrown in /srv/tpl/skytemplate5.php on line 28" while reading response header from upstream, client: 10.90.10.20, server: , request: "GET / HTTP/1.1", upstream: "fastcgi://unix:/run/php-fpm.sock:", host: "skypaste.ch"
10.90.10.20 - - [03/Sep/2024:07:21:22 +0000] "GET / HTTP/1.1" 500 1043 "-" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/128.0.0.0 Safari/537.36" "81.221.X.X"
2024/09/03 07:21:22 [error] 21#21: *5 FastCGI sent in stderr: "PHP message: PHP Fatal error:  Uncaught Error: Call to undefined method PrivateBin\View::_scriptTag() in /srv/tpl/skytemplate5.php:28

Stack trace:
#0 /srv/lib/View.php(58): include()
#1 /srv/lib/Controller.php(439): PrivateBin\View->draw()
#2 /srv/lib/Controller.php(157): PrivateBin\Controller->_view()
#3 /var/www/index.php(17): PrivateBin\Controller->__construct()

}
```
 

### Did you use the FAQ section?

- [X] Yes, I have read [the FAQ](https://github.com/PrivateBin/PrivateBin/wiki/FAQ) and I found no solution/answer there.

### What you did?

1. Copy the Bootstrap5 template. 
2. Transfer the file to the container.
```
    volumes:
      - ./template5.php:/srv/tpl/template5.php:ro
```
3. Change the template in the config file to the template you created.

### Server OS

Debian with Docker

### Webserver

Nginx

### PrivateBin version

1.7.4

### Browser and version

Version 1.69.160 Chromium (Brave Desktop)

### Local operating system and version

Windows 10

### Issue reproducibility

No, I cannot reproduce it on https://privatebin.net.

**Comment by elrido** [comment, marked as answer]
You are trying to use the template from the master branch (unreleased version) with PrivateBin of version 1.7.4 or below. The `_scriptTag` method got introduced only recently (#1365). Download a template from the [release archive](https://github.com/PrivateBin/PrivateBin/releases) matching your container image tag.

**Comment by Skyfay** [comment]
That worked. Thanks for the help!
